src/component/feedback/FeedbackCES/FeedbackCES.tsx

```diff
@@ -0,0 +1,44 @@
+import { Modal } from '@material-ui/core';
+import React, { useContext } from 'react';
+import {
+    feedbackCESContext,
+    IFeedbackCESState,
+} from 'component/feedback/FeedbackCESContext/FeedbackCESContext';
+import { FeedbackCESForm } from 'component/feedback/FeedbackCES/FeedbackCESForm';
+import { useStyles } from 'component/feedback/FeedbackCES/FeedbackCES.styles';
+import { CloseOutlined } from '@material-ui/icons';
+
+export interface IFeedbackCESProps {
+    state?: IFeedbackCESState;
+}
+
+export const FeedbackCES = ({ state }: IFeedbackCESProps) => {
+    const { hideFeedbackCES } = useContext(feedbackCESContext);
+    const styles = useStyles();
+
+    const closeButton = (
+        <button className={styles.close} onClick={hideFeedbackCES}>
+            <CloseOutlined titleAccess="Close" className={styles.closeIcon} />
+        </button>
+    );
+
+    const modalContent = state && (
+        <FeedbackCESForm state={state} onClose={hideFeedbackCES} />
+    );
+
+    return (
+        <Modal
+            open={Boolean(state)}
+            onClose={hideFeedbackCES}
+            disableEscapeKeyDown
```

Comment:
I'm not overly familiar with the Material Modal API, but does this disable closing on pressing `esc`? Is there any reason we're doing that? Can the user close the modal otherwise without submitting feedback?

There seems to be a close button, so `esc` should close the modal, as specified in the [WAI-aria authoring practices](https://w3c.github.io/aria-practices/#keyboard-interaction-7).

Reply:
Yeah, agreed. There was a specific request for `esc` to be disabled, but will remove this for now.